packages/is-shallow-equal/src/index.js

```diff
@@ -13,10 +13,10 @@ export { default as isShallowEqualArrays } from './arrays';
 
 /**
  * Returns true if the two arrays or objects are shallow equal, or false
- * otherwise.
+ * otherwise. Also handles primitives and falsey values, just in case.
```

Comment:
Falsy values are also primitives, aren't they?

```suggestion
 * otherwise. Also handles primitive values, just in case.
```